crates/fuzzing/src/oracles.rs

```diff
@@ -26,6 +29,23 @@ fn fuzz_default_config(strategy: Strategy) -> Config {
     return config;
 }
 
+fn log_wasm(wasm: &[u8]) -> PathBuf {
```

Comment:
Nothing is using the returned `PathBuf`, so I think we don't need to return anything from this function. This also lets us avoid the weird `foo.wasm` phony return when logging is disabled.

Reply:
whoops, I was totally gonna log this path but never got around to doing so...